<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>510</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>3D场景管理主窗口</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QFrame" name="frameProj">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>工程名称</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditProjectName">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>3000</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(231, 231, 231);</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>工程信息</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditProjectInfo">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frameCont">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidgetParts">
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::West</enum>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tabWidgetPageDataSet" native="true">
          <attribute name="title">
           <string>场区数据集</string>
          </attribute>
          <widget class="QPushButton" name="pushButtonAddDataSet">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>131</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>添加场区数据集</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonDelDataSet">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>131</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>删除场区数据集</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonShowDataSet">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>131</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>显示选择数据集</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonShow3DView">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>131</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>打开3D视图</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>180</y>
             <width>81</width>
             <height>81</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="Shader.qrc">:/ZoomIn.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tabWidgetPageSoundSource" native="true">
          <attribute name="title">
           <string>声源</string>
          </attribute>
          <widget class="QPushButton" name="pushButtonAddSource">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>131</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>添加声源数据</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonDelSource">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>131</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>删除声源数据</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonSeeSource">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>131</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>声源居中显示</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tabWidgetPageViewPot" native="true">
          <attribute name="title">
           <string>视景点</string>
          </attribute>
          <widget class="QPushButton" name="pushButtonAddViewPot">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>131</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>添加视景点</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_6">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>131</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>删除视景点</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>131</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>存储当前视景</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonModViewPot">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>131</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>修改视景点</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>170</y>
             <width>131</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>切换到视景点</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidgetParts">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="pageDataSetCont">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableWidget" name="tableWidgetDataSet">
             <column>
              <property name="text">
               <string>√</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>名称</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>类型</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>显示方式</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>经度0(度)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>经度1(度)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>纬度0(度)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>纬度1(度)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>东西(km)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>南北(km)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>深度(米)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>东西点</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>南北点</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>深度点</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>场区信息</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditDataSetInfo">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(231, 231, 231);</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageSoundSrcCont">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableWidget" name="tableWidgetSource">
             <column>
              <property name="text">
               <string>名称</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>经度(度)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>纬度(度)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>深度(米)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>频率(M)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>特性</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageViewPosCont">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableWidget" name="tableWidgetViewPot">
             <column>
              <property name="text">
               <string>视景名称</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>视图类型</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>备注信息</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>视景参数</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFileMan">
    <property name="title">
     <string>文件操作</string>
    </property>
    <widget class="QMenu" name="menuSimu">
     <property name="title">
      <string>创建模拟数据集</string>
     </property>
     <addaction name="actionsimuA"/>
     <addaction name="actionsimuB"/>
     <addaction name="actionsimuC"/>
    </widget>
    <addaction name="menuSimu"/>
    <addaction name="actionOpen"/>
    <addaction name="actionLoad_BSI_DB"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>工具箱</string>
    </property>
    <addaction name="actionColorMapSet"/>
    <addaction name="actionMakeBSIDBs"/>
    <addaction name="actionConfig"/>
    <addaction name="actionTest"/>
    <addaction name="actionTest_2"/>
   </widget>
   <widget class="QMenu" name="menuProjectMan">
    <property name="title">
     <string>工程管理</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="actionSaveAsProject"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionUseMergeCubeField"/>
    <addaction name="actionUseChooseCubeField"/>
    <addaction name="actionExitApp"/>
   </widget>
   <widget class="QMenu" name="menuViewMan">
    <property name="title">
     <string>视图控制</string>
    </property>
    <addaction name="actionViewLookDown"/>
    <addaction name="actionViewLookSide"/>
    <addaction name="actionViewLookIso3D"/>
    <addaction name="separator"/>
    <addaction name="actionBestLookDown"/>
    <addaction name="actionBestLookSide"/>
    <addaction name="actionBestLookIso3D"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelpAbout">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuLayoutMan">
    <property name="title">
     <string>窗口布局</string>
    </property>
    <addaction name="actionSaveLayout1"/>
    <addaction name="actionSaveLayout2"/>
    <addaction name="actionSaveLayout3"/>
    <addaction name="separator"/>
    <addaction name="actionUserLayout1"/>
    <addaction name="actionUserLayout2"/>
    <addaction name="actionUserLayout3"/>
   </widget>
   <addaction name="menuProjectMan"/>
   <addaction name="menuFileMan"/>
   <addaction name="menuViewMan"/>
   <addaction name="menuLayoutMan"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelpAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionColorMapSet">
   <property name="text">
    <string>配色方案管理</string>
   </property>
  </action>
  <action name="actionMakeBSIDBs">
   <property name="text">
    <string>构建地图数据库</string>
   </property>
  </action>
  <action name="actionLoad_BSI_DB">
   <property name="text">
    <string>Load BSI DB</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="text">
    <string>新建工程</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>打开工程</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="text">
    <string>系统参数配置</string>
   </property>
  </action>
  <action name="actionViewLookDown">
   <property name="text">
    <string>切换到俯视图</string>
   </property>
  </action>
  <action name="actionViewLookSide">
   <property name="text">
    <string>切换到侧视图</string>
   </property>
  </action>
  <action name="actionViewLookIso3D">
   <property name="text">
    <string>切换到3D视图</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="text">
    <string>保存工程</string>
   </property>
  </action>
  <action name="actionSaveAsProject">
   <property name="text">
    <string>工程另存为</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionUserLayout1">
   <property name="text">
    <string>恢复窗口布局1</string>
   </property>
  </action>
  <action name="actionUserLayout2">
   <property name="text">
    <string>恢复窗口布局2</string>
   </property>
  </action>
  <action name="actionUserLayout3">
   <property name="text">
    <string>恢复窗口布局3</string>
   </property>
  </action>
  <action name="actionBestLookDown">
   <property name="text">
    <string>最佳俯视视景</string>
   </property>
  </action>
  <action name="actionBestLookSide">
   <property name="text">
    <string>最佳侧视视景</string>
   </property>
  </action>
  <action name="actionBestLookIso3D">
   <property name="text">
    <string>最佳3D视图</string>
   </property>
   <property name="toolTip">
    <string>最佳3D视实景</string>
   </property>
  </action>
  <action name="actionSaveLayout1">
   <property name="text">
    <string>保存窗口布局1</string>
   </property>
  </action>
  <action name="actionSaveLayout2">
   <property name="text">
    <string>保存窗口布局2</string>
   </property>
  </action>
  <action name="actionSaveLayout3">
   <property name="text">
    <string>保存窗口布局3</string>
   </property>
  </action>
  <action name="actionExitApp">
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
  <action name="actionsimuA">
   <property name="text">
    <string>simuA</string>
   </property>
  </action>
  <action name="actionsimuB">
   <property name="text">
    <string>simuB</string>
   </property>
  </action>
  <action name="actionsimuC">
   <property name="text">
    <string>simuC</string>
   </property>
  </action>
  <action name="actionaaaa">
   <property name="text">
    <string>aaaa</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionTest_2">
   <property name="text">
    <string>Test 2</string>
   </property>
  </action>
  <action name="actionUseMergeCubeField">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>采用合并区域作为场区范围</string>
   </property>
  </action>
  <action name="actionUseChooseCubeField">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>选择场区作为场区范围</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Shader.qrc"/>
 </resources>
 <connections/>
</ui>
